### DISCUSSION TITLE
How can I reference custom_fields data in Config Templates?

### DISCUSSION THREAD

**Discussion by penM000** [netbox-community/netbox#13554] [Q&A] [answered]
How do I reference custom_fields in Config Templates?
There is an element (boot-delay) in custom_fields, but nothing appears in the Rendered Config when I specify this.
Thank you for all your help.

> Config Templates
> ```
> {% for interface in device.interfaces.all() %} 
> interface ethernet {{ interface.name }} boot-delay {{ interface.custom_fields }}
> {%endfor %}
> ```
> Actual Output
> ```
> interface ethernet Eth1/1 boot-delay <RestrictedQuerySet [<CustomField: boot-delay>]>
> ```

> Config Templates
> ```
> {% for interface in device.interfaces.all() %} 
> interface ethernet {{ interface.name }} boot-delay {{ interface.custom_fields.boot_delay }}
> {%endfor %}
> ```
> Expected output
> ```
> interface ethernet Eth1/1 boot-delay 30
> ```
> 実際の出力
> ```
> interface ethernet Eth1/1 boot-delay 
> ```

**Comment by candlerb** [comment, marked as answer]
I've not use this recently, but I believe either of these should work:

```
{{ interface.cf["boot_delay"] }}
{{ interface.custom_field_data["boot_delay"] }}
```

**Reply by penM000** [reply]
Both of the above statements worked!
Thank you!
